rosetta-source/src/main/rosetta/observable-asset-fro-func.rosetta

```diff
@@ -17,19 +17,19 @@ func IndexValueObservation: <"Retrieve the values of the supplied index on the s
     // data provider - implementation provides observed value from data source
     inputs:
         observationDate date (1..1)
-        floatingRateOption FloatingRateIndex (1..1)
+        interestRateIndex InterestRateIndex (1..1)
```

Comment:
Attributes in this `fro` namespace should be restricted to `FloatingRateIndex` only

Reply:
Moved to `cdm.observable.asset.func`

Reply:
function moved as suggested